colanderalchemy/schema.py

```diff
@@ -113,6 +114,11 @@ class MyModel(Base):
            method of this instance.
 
            Default: 'ignore'
+        parents\_
+           An ``SQLAlchemySchemaNode`` parent list to avoid relationship circular
+           dependencies and thus prevent infinite recursion. Used internally.
+
+           Default: []
```

Comment:
> Should we be documenting an argument that shouldn't really be used externally?

You're right, probably not.

> Is there any use for this externally?

I actually am already *abusing* this to filter certain relationships. I pass in my `User` model to the `parents_` argument to prevent ColanderAlchemy from adding any relationships to `User` in the schema. I have ideas and code for an alternative way to filter columns and relationships, so the use case I described probably shouldn't be supported.